source/Applications/Advanced/Downsample/Downsample.cs

```diff
@@ -0,0 +1,47 @@
+/*
+This sample shows how to downsample a zivid point cloud.
+*/
+
+using System;
+
+class Program
+{
+    static void Main()
+    {
+        try
+        {
+            var zivid = new Zivid.NET.Application();
+
+            var dataFile =
+                Environment.GetFolderPath(Environment.SpecialFolder.CommonApplicationData) + "/Zivid/Zivid3D.zdf";
+            Console.WriteLine("Reading ZDF frame from file: " + dataFile);
+            var frame = new Zivid.NET.Frame(dataFile);
+
+            Console.WriteLine("Getting point cloud from frame");
+            var pointCloud = frame.PointCloud;
+
+            Console.WriteLine("Size of point cloud before downsampling: " + pointCloud.Size + " data point");
+
+            Console.WriteLine("Downsampling point cloud");
+            var downsampledPointCloud = pointCloud.Downsampled(Zivid.NET.PointCloud.Downsampling.By2x2);
+
+            Console.WriteLine("Size of point cloud after downsampling: " + downsampledPointCloud.Size + " data point");
+
+            Console.WriteLine("Setting up visualization");
+            var visualizer = new Zivid.NET.Visualization.Visualizer();
+
+            Console.WriteLine("Visualizing downsampled point cloud");
+            visualizer.Show(downsampledPointCloud);
+            visualizer.ShowMaximized();
+            visualizer.ResetToFit();
```

Comment:
Make this sample equivalent to C++ one. There you visualize before and after. Our goal should be to have as similar samples as possible between programming languages.

Reply:
Hmm the problem here is that it is challenging to visualize two point clouds with Zivid visualization right? If so, skip it.

Reply:
This is not possible in csharp. One can only do one zivid-visualization in csharp (I talked to Stian and he confirmed this). If we want to have them similar then we must also just vizualize one time in c++ or convert them to some other format to vizualize twice.

Reply:
Then we skip visualizing both.

Reply:
Keep as it is.

Reply:
Should I also skip it in c++ then to make them similar?

Reply:
No, I think it is useful. The SDK should be improved to allow visualization multiple times (multiple point clouds).